features/step-definitions/edit.steps.js

```diff
@@ -0,0 +1,13 @@
+const { Given, When, Then } = require('@wdio/cucumber-framework');
+
+const editPage = require('../pageobjects/edit.page');
+
+Then(/^I should verify its content$/, async () => {

```

Comment:
the same here about hardcoded values, values to check write only in feature files
in steps it should be variables